include/mbedtls/ctr_drbg.h

```diff
@@ -1,10 +1,13 @@
 /**
  * \file ctr_drbg.h
  *
- * \brief CTR_DRBG based on AES-256 (NIST SP 800-90)
+ * \brief    CTR_DRBG is based on AES-256, as defined in <em>NIST SP 800-90A: 
+ *           Recommendation for Random Number Generation Using Deterministic 
+ *           Random Bit Generators</em>.
+ *
  */
-/*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+/*  
```

Comment:
Blocker: Please remove the trailing whitespace.

Reply:
done